source/utilities.tex

```diff
@@ -1512,8 +1512,8 @@
 the class template \tcode{tuple} that can be instantiated with any number
 of arguments. Each template argument specifies
 the type of an element in the \tcode{tuple}.  Consequently, tuples are
-heterogeneous, fixed-size collections of values. An instantiation of \tcode{tuple} with
-two arguments is similar to an instantiation of \tcode{pair} with the same two arguments.
+heterogeneous, fixed-size collections of values. A specialization of \tcode{tuple} with
+two arguments is similar to a specialization of \tcode{pair} with the same two arguments.
```

Comment:
Drive by: can we find a word other than "similar" to avoid confusion with the Core Word of Power(TM)? Or just delete the sentence.